src/react/useTemplateChooser.ts

```diff
@@ -1,24 +1,14 @@
-import { TemplateChooserTemplateChosenCallback, TerrificEvent } from "../types/Terrific";
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AppBridgeNative } from "../AppBridgeNative";
+import { TemplateChooserTemplateChosenCallback } from "../types/Terrific";
 
 type UseTemplateChooserType = {
     openTemplateChooser: (callback: TemplateChooserTemplateChosenCallback) => void;
     closeTemplateChooser: () => void;
 };
 
 export const useTemplateChooser = (): UseTemplateChooserType => {
-    const openTemplateChooser = (callback: TemplateChooserTemplateChosenCallback) => {
-        window.application.connectors.events.components.appBridge.component.onTemplateChooserTemplateChosen = callback;
-
-        const $templateChooser = window.application.sandbox.config.tpl.render("c-templatechooser", {});
-        window.application.connectors.events.notify(null, TerrificEvent.OpenModal, {
-            modifier: "flex",
-            $content: $templateChooser,
-        });
-    };
-
-    const closeTemplateChooser = () => {
-        window.application.connectors.events.notify(null, TerrificEvent.CloseModal, {});
-    };
-
-    return { openTemplateChooser, closeTemplateChooser };
+    const appBridge = new AppBridgeNative();
```

Comment:
```suggestion
    const { openTemplateChooser, closeTemplateChooser } = new AppBridgeNative();
```
&
`return { openTemplateChooser, closeTemplateChooser };`